manifests/profile/kubernetes/firewall.pp

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2019 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# Kubernetes firewall settings
+#
+# This is exactly the same as networking::firewall except that it only
+# manages three chains, and it ignores some rules that we're electing to
+# not have puppet manage.
+class nebula::profile::kubernetes::firewall {
```

Comment:
This should be combined somehow into `networking::firewall`, but I'm not sure what the best way is to do that. I haven't given it much thought because, until this moment, this was a WIP branch.